src/components/personCard/PersonCard.js

```diff
@@ -0,0 +1,28 @@
+import ProfileCircle from "../../components/profileCircle";
+import './style.css'
+
+const PersonCard = ({ person }) => {
+
+    const name = `${person.firstName} ${person.lastName}`
+    const userInitials = name.match(/\b(\w)/g)
+
+    return (
+        <li className="personCard">
+            <ProfileCircle initials={userInitials}/>
+            <div className="nameCohort">
+                <p className="personName">{name}</p>
+                {person.role === 'STUDENT' && <p className="cohortInfo">{`{Specialization}`}, Cohort {person.cohort_id}</p>}
```

Comment:
Where is 'Specialization' coming from? If it should be a property like 'person.specialisation' it can just be wrapped in two curly braces like 'person.cohort_id'